src/mesh/Router.cpp

```diff
@@ -339,41 +364,28 @@ Routing_Error perhapsEncode(MeshPacket *p)
             char compressed_out[Constants_DATA_PAYLOAD_LEN] = {0};
 
             int compressed_len;
-            // compressed_len = unishox2_compress_simple(original_payload, p->decoded.payload.size, compressed_out);
-
-            Serial.print("Original length - ");
-            Serial.println(p->decoded.payload.size);
+            compressed_len = unishox2_compress_simple(original_payload, p->decoded.payload.size, compressed_out);
 
```

Comment:
I think payload.size does not include a \0 (ASCII 0) at the end?  Because \0 does not compress well so will add a couple of bytes reducing overall compression ratio.